compiler/one-cmds/tests/README.txt

```diff
@@ -1,6 +1,8 @@
 one-cmds testing
 ================
 
+NOTE. `one-compiler` should be installed before testing.
+
```

Comment:
@mhs4670go This document does not work without installing `one-compiler-test-*.deb` or `nncc-test-*.tar`. It is the reason I've made #7292. As I said several times, I've installed `one-compiler`, but it does not work.